vendor-extra/ContentPageBundle/src/Controller/ContentController.php

```diff
@@ -48,17 +49,20 @@ public function __invoke(Request $request, string $id) : Response
             )
             ->then(
                 function (ResponseInterface $response) use ($request) : Response {
-                    $dom = FluentDOM::load((string) $response->getBody());
+                    $document = FluentDOM::load((string) $response->getBody());
 
                     $context = [
                         'lang' => $request->getLocale(),
                         'dir' => text_direction($request->getLocale()),
                     ];
 
+                    $event = new CreateContentPageEvent($document, $context);
+                    $this->dispatcher->dispatch($event::NAME, $event);
+
                     return new Response(
                         $this->twig->render(
                             $this->template,
-                            $this->contentHandler->handle($dom->documentElement, $context)
+                            $event->getContext() + ['title' => $event->getTitle(), 'content' => $event->getContent()]
```

Comment:
for reference: `title` is only `<title>`, but not any h1 or similar content in the page, I think

Reply:
It seems like the order of elements in `content` is important, because they will be rendered in order in the HTML page by `text.html.twig`

Reply:
Yep, order of the listener execution is important, so the content header ones are high up https://github.com/libero/browser/blob/a9b3a68c4af2876d2a0e4d8d843d24a66703fbb2/vendor-extra/JatsContentBundle/src/Resources/config/services.xml#L14